docs/docs-content/release-notes/release-notes.md

```diff
@@ -39,6 +41,11 @@ to learn more about this release's new features and improvements.
   point to the new repository. Refer to the [Validator](../automation/palette-cli/commands/validator.md) CLI page
   documentation for more details.
 
+- Due to the removal of GKE Kubernetes patch versions, it's critical you update existing cluster profiles to use the new
+  GKE Kubernetes packs to avoid issues. Active clusters using old GKE Kubernetes pack versions may encounter problems
+  like pods failing to start and scaling issues. We recommend deploying new clusters with the updated GKE cluster
+  profile and migrating workloads.
```

Comment:
I would maybe add this as a warning admonition under the first bullet? Feels critical to the point where we'd want a massive yellow box around it 😅